### DISCUSSION TITLE
Pagination Page Number Not Displayed Correctly

### DISCUSSION THREAD

**Discussion by joesan** [chipzoller/hugo-clarity#238] [General]
First of all, thank you very much for this awesome theme. It is exactly what I was looking for. I have a small issue with the pagination numbering display. As per the theme it should show the pages from 1 to 6, but it does not as seen from the blog below:

https://bigelectrons.com/

Any clues as to what might have gone wrong? I would be thankful for the suggestion.

**Comment by onweru** [comment]
We only show 5 pager items at a time due to responsive issues on mobile. 
<img width="1440" alt="Screen Shot 2021-12-20 at 12 33 50" src="https://user-images.githubusercontent.com/16350351/146745302-551c241d-92ed-4bb8-91f4-44647992ca0f.png">

**Comment by joesan** [comment]
But how come your website neonmirrors.net have this paginated rendering that just shows pages 1,2,3 and then has this ... and then 7?

**Reply by chipzoller** [reply]
It may be a holdover from an earlier version. All the pages are there, obviously, and if you click on page 3 it will expose more pages.
